buildAndReleaseTask/pulumi.ts

```diff
@@ -135,6 +135,24 @@ export async function runPulumi() {
         tl.debug(tl.loc("Debug_PrintingVersion"));
         await tl.exec(toolPath, "version");
 
+        const azureStorageContainer = 
+            tl.getVariable("azure.storage.container") ||
```

Comment:
@tronelius I am working on a PR for #24 which should help with not having to manually map variables and instead simply set them in the build environment as the Pulumi docs show. So these wouldn't be required at all once I have my PR ready to go. I am currently testing it.

Reply:
I just merged my PR for #24. I would recommend you take a look at the `master` branch as you would only need to be concerned with running the appropriate `login` command and no longer need to worry about setting the right env vars in here.